src/vs/workbench/browser/parts/views/customView.ts

```diff
@@ -597,8 +597,18 @@ export class CustomTreeView extends Disposable implements ITreeView {
 	private async doRefresh(elements: ITreeItem[]): Promise<void> {
 		if (this.tree) {
 			this.refreshing = true;
-			await Promise.all(elements.map(element => this.tree.updateChildren(element, true)));
-			elements.map(element => this.tree.rerender(element));
+			const parents: ITreeItem[] = [];
```

Comment:
Should we maybe just use a `Set<ITreeItem>` here?

Reply:
Yes, that is what I wanted. Thank you.